src/pal/src/thread/thread.cpp

```diff
@@ -2430,7 +2433,11 @@ PAL_GetStackBase()
     
     pthread_t thread = pthread_self();
     
+#if HAVE_PTHREAD_ATTR_GET_NP
+    status = pthread_attr_get_np(thread, &attr);
+#else
     status = pthread_getattr_np(thread, &attr);
```

Comment:
This is not guaranteed to exist.  Please add a check_library_exists for it as well, and #else out into an error case.